.cirrus.yml

```diff
@@ -547,7 +560,7 @@ task:
     - git fetch origin master # To set FETCH_HEAD, so that "git merge-base" works.
     - flutter config --no-analytics
     - flutter doctor -v
-    - flutter update-packages
+    - flutter update-packages --offline
```

Comment:
what happens if this commit adds a new dependency/upgrades a version that isn't present locally?

Reply:
The fingerprint checks the pubspec.yaml for their checksum, so it should be a cache miss and repopulate if something changes `echo $OS; grep -r --include=pubspec.yaml 'PUBSPEC CHECKSUM' "$CIRRUS_WORKING_DIR"`

Reply:
Oh yeah...